sphinx_press_theme/static/sphinx_press_theme.css

```diff
@@ -0,0 +1,104 @@
+/*
+ *
+ * Defines default styles specific to Sphinx Press,
+ * on top of VuePress styles
+ *
+ */
+
+@import url('https://fonts.googleapis.com/css?family=Roboto+Mono&display=swap');
```

Comment:
not really happy about including google fonts because they are blocked in china...

Reply:
changed to openfonts